html/captive-portal/lib/captiveportal/PacketFence/Controller/Status.pm

```diff
@@ -96,6 +107,18 @@ sub userIsAuthenticated : Private {
     $c->stash(
         nodes    => \@nodes,
     );
+    if (is_dpsk_enabled($c->profile)) {
+        my @provisioners = $c->profile->provisionerObjects();
+        unless(@provisioners){
```

Comment:
I don't think this is necessary since the foreach will just loop through an empty array without it